### DISCUSSION TITLE
How to use compact mode

### DISCUSSION THREAD

**Discussion by katiaxyz** [rodnansol/spring-configuration-property-documenter#87] [Q&A]
How can I use compact mode? Or can I sort alphabetically the groups?

**Comment by nandorholozsnyak** [comment]
Hello @katiaxyz 

You can check the documentation here about the compact mode: https://rodnansol.github.io/spring-configuration-property-documenter/latest/template-customization.html#template-customizations

You can configure it with the `templateMode` configuration property.

---
Sorting is not (yet) available but it could be a really nice addition.
